app/src/routes/login/components/sso-links.vue

```diff
@@ -103,12 +103,24 @@ const errorFormatted = computed(() => {
 		align-items: center;
 		justify-content: center;
 		width: var(--theme--form--field--input--height);
+		height: var(--theme--form--field--input--height);
 		margin: -$sso-link-border-width;
 		background-color: var(--theme--background-accent);
 		border-radius: var(--theme--border-radius);
 
 		span {
 			--v-icon-size: 28px;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+
+			svg {
+				max-width: 70%;
+				max-height: 70%;
+				width: auto;
+				height: auto;
+				vertical-align: middle;
```

Comment:
No, right noticed, it's currently redundant because the span has a single child, I also found that the width/height isn't impacting much because it's already constrained by inheritance.

Reply:
Take a look on current.